src/structures/Interaction.js

```diff
@@ -269,10 +269,70 @@ class Interaction extends Base {
    * Indicates whether this interaction is a {@link SelectMenuInteraction}.
    * @returns {boolean}
    */
+  isAnySelectMenu() {
+    return InteractionTypes[this.type] === InteractionTypes.MESSAGE_COMPONENT && typeof this.values !== 'undefined';
+  }
+
+  /**
+   * Indicates whether this interaction is a {@link SelectMenuInteraction}
+   * @returns {boolean}
+   * @deprecated Use {@link Interaction#isStringSelect()} instead
+   */
   isSelectMenu() {
+    return this.isStringSelectMenu();
```

Comment:
Please stop pushing changes without testing them. That method doesn't even exist, you called it `isStringSelect()` without the `Menu` suffix.